Create  how_to_setup_podsync_on_synology_nas.md (#487)
mxpv/podsync: docs/how_to_setup_podsync_on_synology_nas.md

# Podsync on Synology NAS Guide

*Written by [@lucasjanin](https://github.com/lucasjanin)*

> This installation podsync on a Synology NAS with SSL and use a standard port 443

Reviewer: ```suggestion
This installs `podsync` on a Synology NAS with SSL and port 443
```

> It requiere to have a domaine with ddns and SSL Certificate

Reviewer: ```suggestion
It requires to have a domain with ddns and an SSL Certificate
```

I'm using a ddns from Synolgy with a SSL Certificate. By chance, my provider doesn't block ports 80 and 443.

1. Open "Package Center" and install "Apache HTTP Server 2.4"
2. In the "Web Station", select the default server, click edit and active "Enable personal website"
3. Create a folder "podsync" in web share using "File Station", the path will be like "/volume1/web/podsync" (where the files will be saved)
4. Create a folder "podsync" in another share using "File Station", the path will be like "/volume1/docker/podsync" (where the config will be saved)

> 5. Create a `config.toml` file in Notepad or whatever editor you want to use and copy it into the above folder.

Reviewer: ```suggestion
5. Create a `config.toml` file in Notepad (or any other editor) and copy it into the above folder.
```
